src/polymer-cli.ts

```diff
@@ -42,6 +43,21 @@ export class PolymerCli {
   }
 
   run(args) {
+    // If the "--version"/"-v" flag is ever present, just print
```

Comment:
It's slightly unsatisfying to manually scan when we have the args parser, but not a huge deal. Maybe we could use a separate args definition and first pass for globals. No change needed now though.